airbyte-cdk/python/airbyte_cdk/sources/declarative/auth/oauth.py

```diff
@@ -49,6 +49,7 @@ class DeclarativeOauth2Authenticator(AbstractOauth2Authenticator, DeclarativeAut
     expires_in_name: Union[InterpolatedString, str] = "expires_in"
     refresh_request_body: Optional[Mapping[str, Any]] = None
     grant_type: Union[InterpolatedString, str] = "refresh_token"
+    refresh_request_body_content_type: ContentType = ContentType.X_WWW_FORM_URLENCODED
```

Comment:
default to x-www-form-urlencoded, but will be set to whatever value is set in the manifest